sql/core/src/test/resources/sql-tests/inputs/operators.sql

```diff
@@ -32,3 +32,11 @@ select 1 - 2;
 select 2 * 5;
 select 5 % 3;
 select pmod(-7, 3);
+
+-- check operator precedence (We follow Oracle operator precedence: https://docs.oracle.com/cd/A87860_01/doc/server.817/a85397/operator.htm#997691)
```

Comment:
The link could be ineffective in the future. Could you also copy the table contents here? Thanks!